### PR TITLE
Make the tutorial text bilingual in the Chimple app

### PR DESCRIPTION
**Pull request by jeevithiesh**

### CODE REVIEW

**Comment by jeevithiesh** [comment]
|PR checklist | Status|
|-- |:--:|
Is code analysis task executed and issues resolved in the Java/Kotlin code? | NA
Is code analysis task executed and issues resolved in the CPP/native code? | NA
Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published? | Yes
Is code commenting done properly for the changes made? | Yes
Is correct variable naming convention followed? | Yes
Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files? | NA
Are the message strings used in the application taken from strings.xml file? | NA
Are the permissions which are not required removed from AndroidManifest.java file? | NA
Is the uses-feature tag used for only must have feature in AndroidManifest.java file? | NA
Are the generated files correctly configured in .gitignore? | NA
Are the release signing configurations removed from the gradle files? | NA
Is commented code removed from all files? | NA
Are updated images compressed (< 100 KB)? | NA
Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments? | Yes
Is size of the OBB correct? | Yes
Is OBB file version correctly set in DownloadExpansionFile.java file? | Yes
Is patch OBB file to be deployed? If yes, then are the settings correct in DownloadExpansionFile.java file? | Yes

**Review by ankitk20 — APPROVED** [review]

**Review by ankitk20 — APPROVED** [review]

File: goa/Resources/res/lang/eng/eng.po [hunk 1]
```diff
@@ -440,7 +513,9 @@ msgid "windchar"
 msgstr "हवा"
 
 msgid "word"
-msgstr "हम एक शब्द बनाने के लिए अलग-अलग अक्षर रख सकते है।\n"
+msgstr ""
+"Inorder to create a word, we can use different letters.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:517]: Space between In and order

File: goa/Resources/res/lang/eng/eng.po [hunk 2]
```diff
@@ -404,7 +475,9 @@ msgid "tree_1qwe"
 msgstr "पेड़"
 
 msgid "verb"
-msgstr "क्रिया एक कार्य दिखाती है।\n"
+msgstr ""
+"A verb shows an action. \n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:479]: No space after full stop

File: goa/Resources/res/lang/eng/eng.po [hunk 3]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table8"
-msgstr "ये 8 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 8.\n\n"
+"ये 8 के गुणन सारणी है।"
 
 msgid "table9"
-msgstr "ये 9 के गुणन सारणी है"
+msgstr ""
+"These are the multiplication tables of 9.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:465]: "This is multiplication table of 9."

File: goa/Resources/res/lang/eng/eng.po [hunk 4]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table7"
-msgstr "ये 7 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 7.\n\n"
+"ये 7 के गुणन सारणी है।"
 
 msgid "table8"
-msgstr "ये 8 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 8.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:460]: "This is multiplication table of 8."

File: goa/Resources/res/lang/eng/eng.po [hunk 5]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table6"
-msgstr "ये 6 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 6.\n\n"
+"ये 6 के गुणन सारणी है।"
 
 msgid "table7"
-msgstr "ये 7 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 7.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:455]: "This is multiplication table of 7."

File: goa/Resources/res/lang/eng/eng.po [hunk 6]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table5"
-msgstr "ये 5 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 5.\n\n"
+"ये 5 के गुणन सारणी है।"
 
 msgid "table6"
-msgstr "ये 6 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 6.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:450]: "This is multiplication table of 6."

File: goa/Resources/res/lang/eng/eng.po [hunk 7]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table4"
-msgstr "ये 4 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 4.\n\n"
+"ये 4 के गुणन सारणी है।"
 
 msgid "table5"
-msgstr "ये 5 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 5.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:445]: "This is multiplication table of 5."

File: goa/Resources/res/lang/eng/eng.po [hunk 8]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table3"
-msgstr "ये 3 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 3.\n\n"
+"ये 3 के गुणन सारणी है।"
 
 msgid "table4"
-msgstr "ये 4 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 4.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:440]: "This is multiplication table of 4."

File: goa/Resources/res/lang/eng/eng.po [hunk 9]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table2"
-msgstr "ये 2 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 2.\n\n"
+"ये 2 के गुणन सारणी है।"
 
 msgid "table3"
-msgstr "ये 3 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 3.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:435]: "This is multiplication table of 3."

File: goa/Resources/res/lang/eng/eng.po [hunk 10]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "table10"
-msgstr "ये 10 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 10.\n\n"
+"ये 10 के गुणन सारणी है।"
 
 msgid "table2"
-msgstr "ये 2 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 2.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:430]: "This is multiplication table of 2."

File: goa/Resources/res/lang/eng/eng.po [hunk 11]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
 msgid "synonym"
-msgstr "समानार्थी शब्द वे शब्द हैं जिनका अर्थ समान है।\n"
+msgstr ""
+"Synonyms are words that have the same meaning. \n\n"
+"समानार्थी शब्द वे शब्द हैं जिनका अर्थ समान है।\n"
 
 msgid "table10"
-msgstr "ये 10 के गुणन सारणी है।"
+msgstr ""
+"These are the multiplication tables of 10.\n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:425]: "This is multiplication table of 10."

File: goa/Resources/res/lang/eng/eng.po [hunk 12]
```diff
@@ -341,58 +380,90 @@ msgstr "बड़ा कदम"
...
+"The answer is 15. \n\n"
 "42 - 27. पहले दाईं ओर कैंडीज घटाएं। आप नहीं कर सकते।\n"
 "इसलिए बाईं ओर से 10 कैंडीज के साथ एक कैंडी बॉक्स उधार लें, अब "
 "आप घटा सकते हैं।  12 - 7 = 5. \n"
 "अब कैंडी बक्से को घटाएं क्योंकि 1 कैंडी बॉक्स में पहले से ही है।"
 
 msgid "synonym"
-msgstr "समानार्थी शब्द वे शब्द हैं जिनका अर्थ समान है।\n"
+msgstr ""
+"Synonyms are words that have the same meaning. \n\n"
```
Comment (ankitk20) [inline goa/Resources/res/lang/eng/eng.po:420]: No space after full stop

**Review by ankitk20 — APPROVED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**